### PR TITLE
Seanquiambao/bloghome

### LINKED ISSUE #1168: Engineering Home Page
image

- Will be at `/engineering`
- Need to migrate all engineering blog articles to `/engineering/blog/*` where `*` is the article name
- Do not need the start or examples route
- Github icon should lead directly to the GitHub repo specified in config

### PR DESCRIPTION
**Pull request by seanquiambao**
image

image

need help with applying navbar to engineering layout

### CODE REVIEW

File: src/components/engineering/Landing.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const Landing = () => {
+  return (
+    <div className="flex flex-row items-center  justify-center w-full min-h-screen">
+      <div className="w-5/12 ml-12">
+        <div className="border-l-8 border-hackathon-blue-100 px-5 text-hackathon-green-300 text-3xl mb-3">
+          <div className="text-6xl text-hackathon-blue-200 mb-4 font-bold">
+            Hackathon Website
+          </div>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```
Comment (shahdivyank) [inline src/components/engineering/Landing.jsx]: create a description for this

File: src/components/engineering/Navigation.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { TABS } from "@/data/engineering/Navigation";
+import { SiGithub } from "react-icons/si";
+import Link from "next/link";
+
+const Navigation = () => {
+  return (
+    <div className="sticky top-0 z-10 flex items-center justify-end bg-hackathon-blue-200 h-20 w-screen text-2xl text-white">
+      {TABS.map((tab, index) => {
+        return (
```
Comment (shahdivyank) [inline src/components/engineering/Navigation.jsx]: implicit return

File: src/components/engineering/What.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+const What = () => {
+  return (
+    <div className="flex flex-row items-center justify-evenly text-white w-full  bg-hackathon-blue-100">
+      <Image src={Vertical} alt="Web Pages" />
+
+      <div className="w-1/3">
+        <div className="text-4xl font-bold">What is Hackathon Website?</div>
+        <div className="text-xl my-5">
+          Lorem ipsum dolorsit amet, consectetur adipiscing elit, sed do eiusmod
```
Comment (shahdivyank) [inline src/components/engineering/What.jsx]: create the text

File: src/components/engineering/What.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+          tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+          veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+          ea commodo consequat.
+        </div>
+        <Link
+          href="www.roblox.com"
```
Comment (shahdivyank) [inline src/components/engineering/What.jsx]: crazy link :sob:

File: src/data/engineering/Navigation.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+export const TABS = [
+  {
+    label: "examples",
+    link: "/engineering",
+  },
+  {
+    label: "start",
+    link: "/engineering",
```
Comment (shahdivyank) [inline src/data/engineering/Navigation.js]: the link is the same?

File: src/components/engineering/What.jsx (hunk 4 above)
Comment (seanquiambao) [reply src/components/engineering/What.jsx]: i thought i removed that omg 💀

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**